### PR TITLE
Logo updates, website is now responsive and minor fix

### PR DESCRIPTION
**Pull request by realslimshanky**
This also follows the wireframe #2

### CODE REVIEW

**Comment by Warlord77** [comment]
Resolve the conflicts

**Comment by realslimshanky** [comment]
@Warlord77 sure. after pydelhi conf.

**Comment by sourabhtk37** [comment]
@realslimshanky It's way past PyDelhiConf. Please update your PR.

**Comment by realslimshanky** [comment]
@sourabhtk37 all done!

**Comment by sourabhtk37** [comment]
Looks great. Thanks @realslimshanky 
@CuriousLearner  Is the logo size okay or should we resize it to give content below more visibility?

File: index.html [hunk 1]
```diff
@@ -10,9 +12,9 @@
 <a href="https://github.com/ILUGD/ILUGD.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
 <body class="page-home">
 	<section>
-			<img class="header-logo" src="assets/img/tux.png">
+			<img class="header-logo" src="assets/img/logo.jpg">
     		<p>Welcome to the India Linux Users Group Delhi</p>
-    		<h1>ILUG-D</h1>
+    		<!--<h1>ILUG-D</h1>-->
```
Comment (realslimshanky) [inline index.html]: This has been fixed in the new commit I added.

File: index.html [hunk 2]
```diff
@@ -10,9 +12,9 @@
...
     	</section>
    	<script src="assets/js/main.js"></script>
```
Comment (realslimshanky) [inline index.html]: Sorry, my bad. It's fixed now!

**Comment by realslimshanky** [comment]
@sourabhtk37 I've fixed the issues mentioned on the review.

**Comment by sourabhtk37** [comment]
There is still additional padding above the links section.

screenshot from 2017-07-26 22-56-45

**Comment by realslimshanky** [comment]
@sourabhtk37 there was a lot of space left at the bottom below the links so I intentionally gave a padding to place the link so that the space occupied looks even. For any further test you can also use this link https://realslimshanky.github.io/ILUG-D.github.io/

**Comment by sourabhtk37** [comment]
I think yes. `30` may seems safe. 
[Here](http://goodui.org/)'s something very useful for building good UX.

**Comment by realslimshanky** [comment]
Great resource. I've updated the padding in the new commit. Thanks for guidance.

**Comment by sourabhtk37** [comment]
All looks good now.

**Pull request merged**